tests/scipy_stats_test.py

```diff
@@ -420,6 +420,23 @@ def args_maker():
                             tol=5e-4)
     self._CompileAndCheck(lax_fun, args_maker)
 
+  @genNamedParametersNArgs(5)
+  def testBetaBinomLogPmf(self, shapes, dtypes):
+    rng = jtu.rand_positive(self.rng())
```

Comment:
Would it be worth testing negative inputs to make sure they correctly return infinity?

Reply:
Ah, good catch.  Are you thinking maybe `rng = jtu.rand_default(self.rng())`?  That is what is used to test the bernoulli and Poisson.

Reply:
Yeah something like that – particularly if the return value in this range is something explicit like `inf`